default/stringtables/ja.txt

```diff
@@ -14721,7 +14781,7 @@ BAD_INDUSTRY_DESC
 −	悪い[[metertype METER_INDUSTRY]]: 75%
 
 AVERAGE_INDUSTRY_DESC
-'''	平均的な[[metertype METER_INDUSTRY]]: 100%'''
+　	平均的な[[metertype METER_INDUSTRY]]: 100%
```

Comment:
For example here.  The triple single quotes make sure the leading whitespace is kept.